source/slang/slang-ir-autodiff-cfg-norm.cpp

```diff
@@ -232,7 +240,7 @@ struct CFGNormalizationPass
                 breakFlagValue,
                 block,
                 afterSplitAfterBlock,
-                afterSplitAfterBlock);
+                afterSplitAfterBlock); 
```

Comment:
Not needed for this PR, but in the future we should all form the habit of self reviewing the PR and remove spurious changes like this to keep the change set clean.